src/gpodder/gtkui/interface/common.py

```diff
@@ -154,7 +154,7 @@ def on_text_changed(editable):
         else:
             return None
 
-    def show_login_dialog(self, title, message, username=None, password=None,
+    def show_login_dialog(self, title, message, root_url=None, username=None, password=None,
```

Comment:
Woops: good catch. This would have been bad.

To test:
 - restore from gPodder, I start with an empty GPODDER_HOME:
   `GPODDER_HOME=/tmp/GPO_HOME bin/gpodder -v`
 - authenticate podcast, I start the test-auth-server and subscribe to it:
   `python2 tools/test-auth-server.py&`